src/status_im/subs.cljs

```diff
@@ -513,6 +513,11 @@
  (fn [multiaccounts]
    (> (count multiaccounts) 1)))
 
+(re-frame/reg-sub
+ :multiaccounts.login/keycard-account?
+ :<- [:multiaccounts/login]
+ (fn [{:keys [keycard-pairing]}] keycard-pairing))
```

Comment:
```suggestion
 (fn [{:keys [keycard-pairing]}] (boolean keycard-pairing)))
```

Reply:
As I wrote the code which uses this sub, this is definitely unnecessary here. I will extract this code to a separate function once have a free time, that's in my todo list, so that we will not need to think about this kinda boolean value...